docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# iexec-voucher-subgraph deployer
+
+FROM node:18
```

Comment:
Why not a more recent version?

Reply:
That's the dockerfile given by Pierre directly. So I did not touched it, could be a 1.0.1 version I guess

Reply:
He already tested that version so I don't think changing the node version rn is the good option